crates/indexer/src/bits/mod.rs

```diff
@@ -4,5 +4,5 @@ pub mod bidder_metadata;
 pub mod edition;
 pub mod get_storefronts;
 pub mod metadata;
+pub mod metadata_uri;
 pub mod store_owner;
-pub mod token_account;
```

Comment:
Why is this removed in this PR?  Is this a diff error, or do we need to move some embedded `TokenAccount` refactor to its own PR?

Reply:
It was added in my PR